index.js

```diff
@@ -5,8 +5,21 @@ const pokemon = require('./pokemon');
 exports.all = pokemon;
 exports.random = uniqueRandomArray(pokemon);
 
-exports.getName = id => {
-	const name = pokemon[id - 1];
+function getLocalizedList(lang) {
+	if (!lang || lang === 'en') {
+		return pokemon;
+	}
+
+	try {
+		return require(`./pokemon_${lang}`);
+	} catch (err) {
+		throw new Error(`Localized list for language code ${lang} does not exist`);
```

Comment:
What about this:
>Localized list for language code '${lang}' does not exist. Pull request welcome: https://github.com/sindresorhus/pokemon

? So people have a direct link they could follow.

I also suggest to put dynamic variables (especially codes) into quotes because it helps reading them. WDYT?